imageregistry/v1/types.go

```diff
@@ -30,6 +31,7 @@ const (
 
 // Config is the configuration object for a registry instance managed by
 // the registry operator
+// +openshift:compatibility-gen:level=1
```

Comment:
level 4, exposed=false and make a note to revisit with the image team as a followup to figure  out if users touch this.

Reply:
@deads2k Looks like `Config` and `ImagePruner` (`imageregistry.operator.openshift.io` group) are exposed:

```sh-session
$ oc api-resources  --api-group=imageregistry.operator.openshift.io
NAME           SHORTNAMES   APIGROUP                              NAMESPACED   KIND
configs                     imageregistry.operator.openshift.io   false        Config
imagepruners                imageregistry.operator.openshift.io   false        ImagePruner

$ oc get configs.imageregistry.operator.openshift.io
NAME      AGE
cluster   46h

$ oc get imagepruners.imageregistry.operator.openshift.io 
NAME      AGE
cluster   46h

$ oc version
Client Version: 4.7.0-0.nightly-2020-11-30-172451
Server Version: 4.7.0-0.nightly-2020-12-04-013308
Kubernetes Version: v1.19.2+ad738ba
```

Reply:
that seems... odd.  ok, thanks.